### DISCUSSION TITLE
@antv/g6 4.5.0  plugins 都无法点击

### DISCUSSION THREAD

**Discussion by Manhand** [antvis/G6#3373] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/20868121/146323094-14b0df04-96d4-409e-b8c1-a3afb8062895.png)

**Comment by Yanyan-Wang** [comment, marked as answer]
你说 minimap？通过拖拽改变视窗

**Reply by Manhand** [reply]
后面发现是图层问题，设置了z-index就好了，感谢
